tutor/src/models/user/menu.js

```diff
@@ -163,7 +166,8 @@ const UserMenu = observable({
 
     if (course) {
       courseId = course.id;
-      ({ isTeacher, primaryRole: { type: menuRole } } = course);
+      isTeacher = course.currentRole;
```

Comment:
This assignment seems... odd?

Reply:
heh, yeah I dunno why it's there, it's not used at all